src/main/paradox/docs/additional-info/faq.md

```diff
@@ -1,3 +1,124 @@
+At the heart of Blue Brain Nexus platform lies Knowledge Graph, that provide knowledge representation to enable FAIR principles at Blue Brain and in neuroscience community.
+Indeed, Knowledge Graph Nexus allow scientists to: 1. register and manage neuroscience relevant entity types; 2. Submit data to the platform and describe their provenance using the W3C PROV model; 3. Search, discover, reuse and derive high-quality neuroscience data generated within and outside the platform for the purpose of driving their own scientific endeavours.
+It is explained in 'Understanding Knowledge Graph' [page](https://bluebrain.github.io/nexus/docs/tutorial/knowledge-graph/index.html).
+
+
+**How Nexus support data governance?**
+
+
+**What has to be configured and how an instance can be integrated with Nexus v1?**
+
+
+**How do I report a bug? What information should I include?**
+
+If you have found a bug while using some of the Nexus services, please create an issue labelled “bug” in the BlueBrain/nexus repo on GitHub [here](https://github.com/BlueBrain/nexus/issues/new?labels=bug). You have to include all the details about your case.
+
+
+**Which support Nexus team provide?**
+
+
+
+## Technical
+
+**How can I configure my project?**
+
+**How to ingest my data into Nexus**
+
+(csv, json, json-ld, no schema needed)
+
+**I uploaded my data, but the system is not working. What can I do?**
+
+If ‘Replicas’ is 0/1 there’s a good chance that docker is still pulling the data.
+Then, if you run docker stack ps nexus you can see if everything is up.
+
+
+**What are the clients I can use with Nexus? What are the requirements to run Nexus locally?**
+
+On macOS and Windows, Docker effectively runs containers inside a VM created by the system hypervisor. Nexus requires at least 2 CPUs and 8 GiB of memory in total. You can increase the limits in Docker settings in the menu *Preferences > Advanced*. Further information [here](https://bluebrain.github.io/nexus/docs/getting-started/running-nexus/docker.html).
+
+
+**Where is the sandbox deployed?**
+
+
+**What is a SHACL schema?**
+
+
+
+**How does SHACL compares with JSON Schema?**
+
+
+
+**Do I need to define SHACL schemas to bring data in?**
+
+
+
+**Where can I find SHACL shapes I can reuse (point to resources, like schema.org)?**
+
+
+
+**What is JSON-LD?**
+
+JSON-LD is a JavaScript Object Notation for Linked Data. It is a set of W3C standards specifications which enable a semantic-preserving data exchange and allows to solve the ambiguity problem including a @context object, where every key is associated with an identifier. In this way, the object properties are linked to contexts in a JSON document to concepts in ontologies. 
+JSON-LD is fully compatible with the RDF data model.
+
```

Comment:
JSON-LD is the main format we use for messages exchange. A JSON-LD payload is then converted to an RDF Graph for validation purposes and for ingestion in the Knowledge Graph.